iOS/DuckDuckGo/Subscription/UserScripts/SubscriptionPagesUseSubscriptionFeature.swift

```diff
@@ -293,7 +294,18 @@ final class DefaultSubscriptionPagesUseSubscriptionFeature: SubscriptionPagesUse
         resetSubscriptionFlow()
 
         struct SubscriptionSelection: Decodable {
+            struct Experiment: Codable {
+                let name: String
+                let cohort: String
+
+                func map(to parameters: inout [String: String]) {
+                    parameters["experimentName"] = name
+                    parameters["experimentCohort"] = cohort
+                }
+            }
```

Comment:
New params we need to store and a convenience method for adding the additional params to a dictionary.